jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.c

```diff
@@ -1815,7 +1815,8 @@ ecma_builtin_string_prototype_object_split (ecma_value_t this_arg, /**< this arg
               ecma_string_t *magic_index_str_p = ecma_get_magic_string (LIT_MAGIC_STRING_INDEX);
               ecma_property_t *index_prop_p = ecma_get_named_property (match_array_obj_p, magic_index_str_p);
 
-              ecma_number_t *index_num_p = ecma_get_number_from_value (index_prop_p->u.named_data_property.value);
+              ecma_value_t index_value = ecma_get_named_data_property_value (index_prop_p);
```

Comment:
You suggested using local variables above, and here you don't like it :) Anyway, I would prefer that myself, but I couldn't do it. Please suggest me a form which is valid according to Jerry formatting guidelines.